specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json

```diff
@@ -17,15 +17,7 @@
       "apim_key": []
     }
   ],
-  "x-ms-parameterized-host": {
-    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
-    "parameters": [
-      {
-        "$ref": "#/parameters/AzureRegion"
```

Comment:
If this type is no longer referenced please be sure to remove it from the spec (same comment applied to other swaggers).

Reply:
@jhendrixMSFT I've checked, this seems to be the only place with not used type. (others refer to a external file which is being used by some legacy files)